exporter/opentelemetry-exporter-prometheus/tests/test_prometheus_exporter.py

```diff
@@ -49,31 +56,51 @@ def test_constructor(self):
             self.assertTrue(self._mock_registry_register.called)
 
     def test_shutdown(self):
-        with mock.patch(
+        with patch(
             "prometheus_client.core.REGISTRY.unregister"
         ) as registry_unregister_patch:
             exporter = PrometheusMetricReader()
             exporter.shutdown()
             self.assertTrue(registry_unregister_patch.called)
 
     def test_histogram_to_prometheus(self):
-        record = _generate_metric(
-            "test@name",
-            Histogram(
-                aggregation_temporality=AggregationTemporality.CUMULATIVE,
-                bucket_counts=[1, 3, 2],
-                explicit_bounds=[123.0, 456.0],
-                start_time_unix_nano=1641946016139533244,
-                max=457,
-                min=1,
-                sum=579.0,
-                time_unix_nano=1641946016139533244,
+        metric = Metric(
+            name="test@name",
+            description="foo",
+            unit="s",
+            data=Histogram(
+                data_points=[
+                    HistogramDataPoint(
+                        attributes={"histo": 1},
+                        start_time_unix_nano=1641946016139533244,
+                        time_unix_nano=1641946016139533244,
+                        count=6,
+                        sum=579.0,
+                        bucket_counts=[1, 3, 2],
+                        explicit_bounds=[123.0, 456.0],
+                        min=1,
+                        max=457,
+                    )
+                ],
+                aggregation_temporality=AggregationTemporality.DELTA,
             ),
-            attributes={"histo": 1},
+        )
+        metrics_data = MetricsData(
+            resource_metrics=[
+                ResourceMetrics(
+                    resource=Mock(),
+                    scope_metrics=[
+                        ScopeMetrics(
+                            scope=Mock(), metrics=[metric], schema_url=Mock()
```

Comment:
Might actually be easier to stick real objects in here instead of Mocks since `schema_url` is a string and on L97 😄

Reply:
Fixed